src/BootstrapMixin.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import styleMaps from './styleMaps';
-import CustomPropTypes from './utils/CustomPropTypes';
+import { keyOf } from 'react-prop-types';
```

Comment:
My thought was that we use all of them so it wouldn't make a difference, but as the projects diverge I suppose that does make sense!